include/multipass/cli/prompters.h

```diff
@@ -88,6 +90,24 @@ class NewPassphrasePrompter : public PassphrasePrompter
 
     std::string prompt(const std::string& text = "Please re-enter passphrase") const override;
 };
+
+class BridgePrompter : private DisabledCopyMove
+{
+public:
+    explicit BridgePrompter(Terminal* term) : term(term){};
+
+    ~BridgePrompter() = default;
+
+    bool bridge_prompt(std::vector<std::string>& nets_need_bridging) const;
```

Comment:
Indeed, it could be `const std::vector<std::string>& nets_need_bridging`, btw, unlike the `operator [] `of `map` and `unordered_map`, the `operator []` of `std:: vector `has a const version of that. So using it should have nothing to do with the constness of the std::vector.

Reply:
Ah yes, right I was looking at this as a map.

Reply:
Done, added `const`.